README.md

```diff
@@ -1,2 +1,14 @@
-# hc-http-gw
+# Holochain HTTP Gateway
+
 The Holochain HTTP Gateway for providing a way to bridge from the web2 world into Holochain
+
+## License
+
+[![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
+
+Copyright (C) 2020-2025, Holochain Foundation
+
+This program is free software: you can redistribute it and/or modify it under the terms of the license
+provided in the LICENSE file (CAL-1.0). This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE.
```

Comment:
```suggestion
```

Not CAL licensed. The gateway has to at least ephemerally handle crypto keys for signing but it I believe it does not hold identification keys in the sense that the CAL is designed to cover.